urlconverter.js

```diff
@@ -0,0 +1,27 @@
+var url = require("url");
+var moment = require("moment-timezone");
```

Comment:
Do this to save changes for update package.json
$ npm install moment-timezone --save